codebuild/bin/criterion_baseline.sh

```diff
@@ -35,21 +35,33 @@ upload_artifacts(){
 }
 
 if [ -d "third-party-src" ]; then
-  # Don't run against c.a.c.
+  echo "Not running against c.a.c."
   return 0
 fi
 
-# Fetch creds and the latest release number.
-gh_login s2n_codebuild_PRs
-get_latest_release
-AWS_S3_PATH="integv2criterion_${INTEGV2_TEST}_${LATEST_RELEASE_VER}.zip"
+# setting LOCAL_TESTING disables a check for an existing baseline.
+if [ -z "${LOCAL_TESTING:-}" ]; then
+  # Fetch creds and the latest release number.
+  gh_login s2n_codebuild_PRs
+  get_latest_release
+  # Build a specific filename for this release
+  AWS_S3_PATH="integv2criterion_${INTEGV2_TEST}_${LATEST_RELEASE_VER}.zip"
+  zip_count=$(aws s3 ls "${AWS_S3_URL}${AWS_S3_PATH}"|wc -l||true)
 
-zip_count=$(aws s3 ls "${AWS_S3_URL}${AWS_S3_PATH}"|wc -l||true)
-if [ "$zip_count" -eq 0 ]; then
-  echo "File ${AWS_S3_URL}${AWS_S3_PATH} not found"
-  install_deps
-  TOX_TEST_NAME=${INTEGV2_TEST}.py make integrationv2
-  upload_artifacts
+  # Only do the baseline if an artifact for the current release doesn't exist.
+  if [ "$zip_count" -eq 0 ]; then
```

Comment:
So is the plan that Criterion only runs once per release, not once per PR?

Reply:
Baseline runes once per release, correct- the delta script could be run against a PR or on a schedule.